prijateli_tree/app/templates/ready_to_play.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+{% block title %}
+  PrijateliTree - Game Start
+{% endblock title %}
+{% block content}
+  <h1>Get Ready to Play</h1>
+  <h2 style="color:red;">Red</h2>
+  <h3>or</h3>
+  <h2 style="color:blue;">Blue!</h2>
+  <form class="row" method="post" action="{{ url_for("confirm_ready") }}">
+    <button class="btn icon-right waves-effect waves-light"
+            type="submit"
+            value="submit">
+      I'm Ready!<i class="material-icons right">send</i>
```

Comment:
This text should be taken in from the json documents that are added in this code segment in `main.py`:
```python
languages = {}
for lang in glob.glob("languages/*.json"):
    lang_code = lang.split("\\")[1].split(".")[0]

    with open(lang, FILE_MODE_READ, encoding=STANDARD_ENCODING) as file:
        languages[lang_code] = json.load(file)
```